test/intl402/NumberFormat/default-minimum-singificant-digits.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Tests that the default value of minimumSignificantDigits is 1.
+---*/
```

Comment:
it would be great to add the esid tag here so we have a proper reference to the spec.